CHANGELOG.md

```diff
@@ -3,7 +3,37 @@
 All notable changes to this project will be documented in this file.
 This project adheres to Semantic Versioning.
 
-All changes mention the author, unless contributed by me (@derekparker).
+## [1.4.1] 2020-05-22
+
+### Added
```

Comment:
Can we add `Support linux/386` (#1884)? As I understand, this is the first time we support 32-bit os.